src/helper.ts

```diff
@@ -0,0 +1,237 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as gax from 'google-gax';
+import {Descriptors, ClientOptions} from 'google-gax';
+import * as path from 'path';
+import * as gapicConfig from './iam_policy_service_client_config.json';
+import {ProjectIdCallback} from 'google-auth-library';
+import * as protosTypes from '../protos/protos';
+const version = require('../../package.json').version;
+/**
+ *  Google Cloud IAM Client.
+ *  This is manually written for providing methods [setIamPolicy, getIamPolicy, testIamPerssion] to the KMS client.
```

Comment:
@alexander-fenster I remain scared by this file.  This looks like it was generated by the generated, and hand dropped.  What happens when we make broad changes like updating the list of headers, or fixing the dangling auth promise in constructors?  

@bcoe

Reply:
@xiaozhenliu-gg5 I think the right answer here would be to make IAM client a part of GAX. We already have Operations client there to support LRO across all libraries, and it's probably a good place for IAM client as well.

Let's do this:
- release this as is (with comments as @JustinBeckwith asked)
- in an issue you filed, make a note that we might want to move IAM client implementation to `google-gax`, and provide some API for clients that need it to use it.

Reply:
cool, thanks for the clarification, Alex.